docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1911 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #56.        for i in np.flatnonzero(T_subseq_isfinite):
</pre><p>If you use <code>M_T</code>  to get <code>T_subseqs_isfinite</code>  then you don't need the <code>np.flatnonzero</code>  anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
My mistake. I misinterpreted `np.flatnonzero` and forgot that it returns the indices where elements are `True`. Please ignore